ipatests/test_webui/ui_driver.py

```diff
@@ -1163,6 +1167,24 @@ def navigate_to_row_record(self, row, pkey_column=None):
         self.wait_for_request(0.4)
         self.wait_for_request()
 
+    def navigate_to_row_record_in_new_tab(self, row, pkey_column=None):
+        """
+        Navigate to record in a new tab by CTRL + SHIFT clicking on the link
+        which brings target focused in a new tab.
+        """
+        s = 'a'
+        if pkey_column:
+            s = "div[name='%s'] a" % pkey_column
+        link = self.find(s, By.CSS_SELECTOR, row, strict=True)
+        action_driver = ActionChains(self.driver)
+        action_driver.key_down(
+            Keys.CONTROL).key_down(
```

Comment:
There can be a more convenient formatting, like:

```python
        (action_driver.key_down(Keys.CONTROL)
                      .key_down(Keys.SHIFT)
                      .click(link)
                      .key_up(Keys.CONTROL)
                      .key_up(Keys.SHIFT)
                      .perform())
```
or some variation of.

Reply:
You are right, it's much more readable. Fixed, thank you